libraries/botbuilder-applicationinsights/src/applicationInsightsTelemetryClient.ts

```diff
@@ -74,16 +74,34 @@ export class ApplicationInsightsTelemetryClient implements BotTelemetryClient, B
     private config: appInsights.Configuration;
 
     /**
-     * Creates a new instance of the [ApplicationInsightsTelemetryClient](xref:botbuilder-applicationinsights.ApplicationInsightsTelemetryClient) class.
+     * Creates a new instance of the
+     * [ApplicationInsightsTelemetryClient](xref:botbuilder-applicationinsights.ApplicationInsightsTelemetryClient)
+     * class.
+     *
+     * @param connectionString The ApplicationInsights connection string.
```

Comment:
It would be beneficial for users to highlight the differences between passing in a connection string versus passing in the instrumentation key.

Reply:
I can't find any good docs to link to at the moment, so I created https://github.com/microsoft/botbuilder-js/issues/3552 to track this.